tests.yaml

```diff
@@ -77,24 +77,48 @@ tests:
   bootrr:
     title: Bootrr test suite
     home: https://github.com/andersson/bootrr
+  bpf:
+    title: In-tree BPF test suite module
+    home: https://git.kernel.org/pub/scm/linux/kernel/git/torvalds/linux.git/tree/lib/test_bpf.c
   dacapobench:
     title: DaCapo Java benchmarking suite
     home: http://dacapobench.sourceforge.net/
   fio:
     title: Flexible I/O tester
     home: https://github.com/axboe/fio
+  fwts:
```

Comment:
Arguably not a Linux kernel test suite, but it exercises the kernel so might be fine in this respect.

The results are very likely to be tied to the firmware revision present on the DUT though, rather than the kernel, which can complicate things a bit.

Reply:
Yeah, this is borderline, but if we see it is useless (i.e. unreliable) we can just stop reporting it and drop it from the catalog. So far we're running it for RHEL only, but we can likely run it for upstream as well.

Reply:
OK - also there have been discussions in the past about using KernelCI infrastructure to test firmwares and bootloaders, in which case the kernel would be a known good one and not a moving part.  I guess it's benign to have `fwts` in the catalog as long as we all know what to expect from this kind of tests.